site/src/pages/AuditPage/AuditLogRow/AuditLogDescription/AuditLogDescription.tsx

```diff
@@ -57,3 +60,19 @@ export const AuditLogDescription: FC<AuditLogDescriptionProps> = ({
 		</span>
 	);
 };
+
+function AppSessionAuditLogDescription({ auditLog }: AuditLogDescriptionProps) {
+	const { connection_type, workspace_owner, workspace_name } =
+		auditLog.additional_fields;
+
+	return (
+		<>
+			{connection_type} session to {workspace_owner}'s{" "}
+			<Link component={RouterLink} to={`@${workspace_owner}/${workspace_name}`}>
```

Comment:
```suggestion
			<Link component={RouterLink} to={`${auditLog.resource_link}`}>
```
This will be fixed by #16670.